### PR TITLE
MONA-93 - Add New Slide Panel Component

### PR DESCRIPTION
**Pull request by mcastre**

### CODE REVIEW

File: style-guide/templates/sg-slide-panel-template.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="close()">
+    <span class="sr-only">Close</span>
+  </button>
+  <h4 class="modal-title">{{panelData.panelTitle}}</h4>
```
Comment (mstone6769) [inline style-guide/templates/sg-slide-panel-template.html]: using an `h4` willy nilly 

probably should be an `h2`

File: style-guide/templates/sg-slide-panel-template.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  <p>You can add a slide panel description or further instructions here.</p>
+</div>
+<div class="modal-body">
+  <p>Slide panel content goes here.</p>
+</div>
+<ul class="list-group">
```
Comment (mstone6769) [inline style-guide/templates/sg-slide-panel-template.html]: a list group should be inside the modal body

File: style-guide/templates/sg-slide-panel-template.html (hunk 1 above)
Comment (mcastre) [inline style-guide/templates/sg-slide-panel-template.html]: good call. Dont know where I got the h4 from

File: style-guide/templates/sg-slide-panel-template.html (hunk 2 above)
Comment (mcastre) [inline style-guide/templates/sg-slide-panel-template.html]: List group is good to have on the outside so that the lines be flush from left to right

**Pull request merged**